timesketch/ui/templates/sketch/timelines.html

```diff
@@ -20,40 +20,42 @@
 <div class="container">
 
   {% if sketch.has_permission(current_user, 'write') %}
-  <div class="row">
-    <div class="col-md-12">
-      <form method="get" action="{{ url_for('sketch_views.timelines', sketch_id=sketch.id) }}">
-        <input type="text" name="q" class="search-input" placeholder="Search for timelines to add" />
-      </form>
-    </div>
-  </div>
-  <br>
+      <div class="row">
+        <div class="col-md-12">
+          <form method="get" action="{{ url_for('sketch_views.timelines', sketch_id=sketch.id) }}">
+            <input type="text" name="q" class="search-input" placeholder="Search for timelines to add" />
+          </form>
+        </div>
+      </div>
+      <br>
 
-  {% if form.timelines.choices %}
-  <div class="row">
-    <div class="col-md-12">
-      <div class="card">
-        <form method="post" action="{{ url_for('sketch_views.timelines', sketch_id=sketch.id) }}">
-          {{ form.timelines }}
-          {{ form.csrf_token }}
-          {% if form.timelines.choices %}
-          <button class="btn btn-primary" type="submit">Add to sketch</button>
-          {% endif %}
-        </form>
+      {% if form.timelines.choices %}
+          <div class="row">
+            <div class="col-md-12">
+              <div class="card">
+                <h4>Last 20 timelines added</h4>
```

Comment:
I think it's better to not hardcode this number - pull it out into a var somewhere else.